UnityProject/Assets/Scripts/Health/CPRable.cs

```diff
@@ -64,21 +64,33 @@ private void ServerDoCPR(GameObject performer, GameObject target,BodyPartType Ta
 
 		health.CirculatorySystem.ConvertUsedBlood(OXYLOSS_HEAL_AMOUNT);
 
+		bool HasHeart = false;
 		foreach (var BodyPart in health.GetBodyPartsInZone(TargetBodyPart))
 		{
 			var heart = BodyPart as Heart;
 			if (heart != null)
 			{
 				heart.Heartbeat(HeartbeatStrength);
+				HasHeart = true;
 			}
-		}
-		health.GetBodyPartsInZone(TargetBodyPart);
 
+		}
 
-		Chat.AddActionMsgToChat(
+		if (HasHeart)
+		{
+			Chat.AddActionMsgToChat(
 				performer,
 				$"You perform CPR on {targetName}.",
 				$"{performerName} performs CPR on {targetName}.");
-		Chat.AddExamineMsgFromServer(target, $"You feel fresh air enter your lungs. It feels good!");
+			Chat.AddExamineMsgFromServer(target, $"You feel fresh air enter your lungs. It feels good!");
+		}
+		else
+		{
+			Chat.AddActionMsgToChat(
+				performer,
+				$"You perform CPR on {targetName} Finding it doesn't help Maybe they're missing something.",
```

Comment:
this could be rephrased, I think

Reply:
what to?

Reply:
> Finding it doesn't help

I don't understand this part, is it me not knowing English?

Reply:
finding what?

Reply:
It is this case would be the CPR : "You perform CPR on Bob, finding the CPR doesn't help."

Reply:
"You perform CPR on {targetName}. It doesn't seem to work, maybe they're missing something."

Reply:
absolutely, that sounds a lot better in my peepeepoopoo non-anglo head.

Reply:
yeah, need to add a , will update with suggested one